private_sharing/tests.py

```diff
@@ -458,6 +488,11 @@ class SmokeTests(SmokeTestCase):
         '/direct-sharing/projects/on-site/create/',
         '/direct-sharing/projects/on-site/update/abc-2/',
         '/direct-sharing/projects/on-site/abc-2/',
+
+        # Tests desired appear to fail due to missing 'css_classes', probably
```

Comment:
https://docs.djangoproject.com/en/2.1/ref/forms/api/#django.forms.BoundField.css_classes

Seems to be still supported?

Reply:
Maybe how it's done shifted? I really don't know. I couldn't easily see how to make the test not fail, unfortunately, but I left this here in the hopes that someone else might solve it. :-/

Reply:
I'll add it to my todo list under the tests section ...